### PR TITLE
moving ArgoCD

### PR DESCRIPTION
**Pull request by kalexand-rh**
Per @aheslin

### CODE REVIEW

File: _topic_map.yml [hunk 1]
```diff
@@ -105,6 +105,8 @@ Topics:
 - Name: The CI/CD methodology and practice
   File: cicd_gitops
   Distros: openshift-enterprise,openshift-webscale
+- Name: Using ArgoCD
```
Comment (bergerhoffer) [inline _topic_map.yml:108]: I was going to say that I didn't see how this fits under Architecture, but I do see that there is already a CI/CD assembly here, so I guess it makes sense to keep those together.

**Review by bergerhoffer — COMMENTED** [review]
Just one comment, and for some reason the preview doesn't seem to be showing it under Architecture even though I see that's where it is in the topic map. So I'm not sure what to say about that.

File: _topic_map.yml (hunk 1 above)
Comment (kalexand-rh) [reply _topic_map.yml:108]: > Just one comment, and for some reason the preview doesn't seem to be showing it under Architecture even though I see that's where it is in the topic map. So I'm not sure what to say about that.

It's rendering correctly in the Netlify and local preview for me, so I'm not sure what's gone wrong.

> I was going to say that I didn't see how this fits under Architecture, but I do see that there is already a CI/CD assembly here, so I guess it makes sense to keep those together.

That was Ali's reasoning, too. I'd be happier to remove the topic, but /shrug.

Comment (bergerhoffer) [reply _topic_map.yml:108]: The netlify preview now looks good to me. Not sure why it wasn't right when I looked at it an hour ago.

**Pull request merged**